src/lib.rs

```diff
@@ -428,7 +427,10 @@ fn iter_config_paths(font_dir: &Path) -> Result<impl Iterator<Item = PathBuf>, C
         else {
             return false;
         };
-        stem.starts_with("config") && (extension == "yaml" || extension == "yml")
+        !stem.starts_with("fontbakery")
```

Comment:
here's an example of a fontbakery configuration file that we want to avoid in the gfsources results: https://github.com/notofonts/imperial-aramaic/blob/main/fontbakery.yml